packages/main/src/models/generative-model.ts

```diff
@@ -157,7 +158,8 @@ export class GenerativeModel {
   async countTokens(
     request: CountTokensRequest | string | Array<string | Part>,
   ): Promise<CountTokensResponse> {
-    const formattedParams = formatGenerateContentInput(request);
+    const formattedParams = formatCountTokensInput(request, this.model);
+    // countTokens() appends the model to the CountTokensRequest proto.
```

Comment:
I think this comment should be for the previous line? `countTokens` doesn't append anything. It uses the model param to build the request url.

Reply:
It adds it to the request body, but I'll remove the comment.

Reply:
countTokens() doesn't, but formatCountTokensInput() does, right? That's why I think it would be best to attach it to the previous line? countTokens() just calls makeRequest() and returns response.json().

Reply:
countTokens() also adds it to the proto request object [here](https://github.com/google-gemini/generative-ai-js/blob/9318ee8d5aa0d15a65039af37a9d972af0739b10/packages/main/src/methods/count-tokens.ts#L36).

Reply:
Hm, that seems to have been there since initial commit but I can't remember why it was needed. I don't know if countTokens is special or if I forgot to remove the model there when I removed it from generateContent and the other methods, let me double check. If it's just left over from the beginning we should probably remove it to avoid unexpected behavior.

Reply:
I don't think this blocks this PR though.